shacl12-core/index.html

```diff
@@ -1752,6 +1760,10 @@ <h4>Declaring the Severity of a Shape or Constraint</h4>
 							<td>A constraint violation.</td>
 						</tr>
 					</table>
+					<p>
+						Validation engines may treat <code>sh:Info</code> and <code>sh:Warning</code> as non-violating based on
+						options passed to the engine. By default, they must be treated as violations.
```

Comment:
... they are treated as violations.

(This avoids the special semantics of "must")